context.c

```diff
@@ -430,3 +438,22 @@ struct Mailbox *ctx_mailbox(struct Context *ctx)
 {
   return ctx ? ctx->mailbox : NULL;
 }
+
+/**
+ * ctx_set_mailbox - Handle the post-mx_mbox_open steps
+ * @param m Mailbox
+ * TODO - kill this
+ */
+struct Context *ctx_set_mailbox(struct Mailbox *m)
```

Comment:
Is this needed any more?

Reply:
No, I haven't polished the branch yet, don't waste time on the details :)